app/_data/tables/version_errors_konnect.yml

```
@@ -2022,157 +2228,25 @@ messages:
-      intended.
-    Resolution: |
-      Please upgrade Kong Gateway to version `2.7.0.0` or above.
-    DocumentationURL: |
-      [IP Restriction plugin](/hub/kong-inc/ip-restriction/)
-  - ID: P108
-    Severity: error
-    Description: |
-      For the `rate-limiting` plugin, one or more of the following `config` fields are
-      set: `redis_ssl`, `redis_ssl_verify`, `redis_server_name` but Kong Gateway
-      versions < 2.7.0.0 do not support these fields. Plugin features that rely on
-      these fields are not working as intended.
-    Resolution: |
-      Please upgrade Kong Gateway to version `2.7.0.0` or above.
-    DocumentationURL: |
-      [Rate Limiting plugin](/hub/kong-inc/rate-limiting/)
-  - ID: P109
-    Severity: error
-    Description: |
-      For the `zipkin` plugin, one or more of the following fields: `header_type` have
-      been set to one or more of the following values: `ignore` but Kong Gateway
-      versions < 2.7.0.0 do not support these values. Impacted fields have been
-      updated to the following default values in the data plane node: `preserve`.
-      Plugin features that rely on these values may not be working as intended.
-    Resolution: |
-      Please upgrade Kong Gateway to version `2.7.0.0` or above.
-    DocumentationURL: |
-      [Zipkin plugin](/hub/kong-inc/zipkin/)
-  - ID: P110
-    Severity: error
-    Description: |
-      For the `zipkin` plugin, one or more of the following `config` fields are set:
-      `local_service_name` but Kong Gateway versions < 2.7.0.0 do not support these
-      fields. Plugin features that rely on these fields are not working as intended.
-    Resolution: |
-      Please upgrade Kong Gateway to version `2.7.0.0` or above.
-    DocumentationURL: |
-      [Zipkin plugin](/hub/kong-inc/zipkin/)
-  - ID: P111
-    Severity: error
-    Description: |
-      For the `acme` plugin, one or more of the following `config` fields are set:
-      `rsa_key_size` but Kong Gateway versions < 2.8.0.0 do not support these fields.
-      Plugin features that rely on these fields are not working as intended.
-    Resolution: |
-      Please upgrade Kong Gateway to version `2.8.0.0` or above.
-    DocumentationURL: |
-      [ACME plugin](/hub/kong-inc/acme/)
-  - ID: P112
+  - ID: EOL107
     Severity: error
     Description: |
-      For the `rate-limiting` plugin, one or more of the following `config` fields are
-      set: `redis_username`, `redis.username` but Kong Gateway versions < 2.8.0.0 do
-      not support these fields. Plugin features that rely on these fields are not
-      working as intended.
+      Kong Gateway version 3.5.x.x has now reached its End of Full Support, and after
+      Dec 1, 2024, data plane connections for this version will no longer receive
+      configurations from Konnect.
     Resolution: |
-      Please upgrade Kong Gateway to version `2.8.0.0` or above.
+      To ensure continued Konnect support, please upgrade to the latest 3.9 version.
```

```suggestion
      To ensure continued Konnect support, please upgrade to the latest Gateway version.
```
Should this say 3.9? 3.10 is out now, and in general you probably just want it to say latest.